app/views/pages/governance/bida.njk

```diff
@@ -57,13 +57,18 @@
             <li>As instructed, other programmes being delivered across national Health and Social Care bodies where the focus is on national capability or where there is a clear interface between national and local capabilities.</li>
           </ul>
 
-
+          <figure class="nhsuk-image">
+            <img class="nhsuk-image__img" src="/images/trg.jpg" alt="Picture of the TRG and EAB process flow">
```

Comment:
@AdamChrimes this image URL is not showing, the image is located in the app/assets/images/